R/annotate_sv.R

```diff
@@ -52,7 +52,7 @@ annotate_sv = function(sv_data,
                        with_chr_prefix = FALSE,
                        collapse_redundant = FALSE,
                        return_as = "bedpe",
-                       blacklist = c(60565248, 30303126, 187728894, 101357565, 101359747, 161734970, 69400840, 65217851, 187728889, 187728888,187728892, 187728893,188305164),
+                       blacklist = c(60565248, 30303126, 187728894, 101357565, 101359747, 161734970, 69400840, 65217851, 187728889, 187728888,187728892, 187728893,188305164, 72551424L, 72551425L, 72551554L, 72551555L, 72551558L, 72551559L, 72551562L, 189255425L, 189255426L, 189255461L, 189255462L),
```

Comment:
Is there a problem with interpreting these numbers in blacklist as integers? Does a mix-up of explicit integers and not makes any difference? Maybe it is worth using a consistent way here, either denote everything as integer if there is any issue with interpretation, or do not denote half as integer and half as not. This way, at least the message is clear of what is expected here and what the user should supply if they want to go with non-default values

Reply:
done